test/gtest/mp/CMakeLists.txt

```diff
@@ -107,16 +107,16 @@ if(ENABLE_SYCL)
 
   foreach(nproc RANGE 1 4)
     add_mp_ctest(NAME mp-quick-test NPROC ${nproc} SYCL)
-    add_mp_ctest(
+    add_mp_disabled_ctest(
```

Comment:
this is a quick workaround

it works for now as we test only on github public runners, however it causes these tests not to run on PVC too if we will want to run them manually

therefore I proposed doing some magic with two separate tags to have set of tests
- good to run on gitghub (all enabled except OFFLOAD ones)
- good to run on PVC (all enabled)

I'd like to have a way to run by hand all previously enabled tests on PVC by one command. Now I can't because good for PVC tests are disabled togerther with e.g. gdb tests which are usually not good to run.

Could you please fix it?
I agree it can be fixed in some next PR after merging this one if we need workaround quickly.

Reply:
Added additional label for running pvc specific tests: PVCLABEL